test_tkinter and test_ttk failed on x86 Debian Non-Debug with X 3.x

Build: https://buildbot.python.org/#/builders/1245/builds/8893

test.pythoninfo:

```
tkinter.TCL_VERSION: 8.6
tkinter.TK_VERSION: 8.6
tkinter.info_patchlevel: 8.6.13
```

test_ttk has multiple `test_configure_padding()` failures. One example (the other ones are similar):

```
======================================================================
FAIL: test_configure_padding (test.test_ttk.test_extensions.OptionMenuTest.test_configure_padding)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_ttk/test_widgets.py", line 38, in test_configure_padding
    self.checkParam(widget, 'padding', 0, expected=(0,), conv=padding_conv)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_tkinter/widget_tests.py", line 77, in checkParam
    self.assertEqual2(widget[name], expected, eq=eq)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_tkinter/widget_tests.py", line 50, in assertEqual2
    self.assertEqual(actual, expected, msg)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Tuples differ: (0,) != ('0',)

First differing element 0:
0
'0'

- (0,)
+ ('0',)
?  + +
```

test_tkinter failure:

```
======================================================================
FAIL: test_configure_tabs (test.test_tkinter.test_widgets.TextTest.test_configure_tabs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_tkinter/test_widgets.py", line 917, in test_configure_tabs
    self.checkParam(widget, 'tabs', '10.2 20.7 1i 2i',
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                    expected=(10.2, 20.7, '1i', '2i')
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                             if get_tk_patchlevel(self.root) >= (8, 6, 14)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                             else ('10.2', '20.7', '1i', '2i'))
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_tkinter/widget_tests.py", line 77, in checkParam
    self.assertEqual2(widget[name], expected, eq=eq)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildbot/buildarea/3.x.ware-debian-x86.nondebug/build/Lib/test/test_tkinter/widget_tests.py", line 50, in assertEqual2
    self.assertEqual(actual, expected, msg)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Tuples differ: (10.2, 20.7, <pixel object: '1i'>, <pixel object: '2i'>) != ('10.2', '20.7', '1i', '2i')

First differing element 0:
10.2
'10.2'

- (10.2, 20.7, <pixel object: '1i'>, <pixel object: '2i'>)
+ ('10.2', '20.7', '1i', '2i')
```

cc @serhiy-storchaka @zware @terryjreedy

--- a/Lib/test/test_tkinter/test_widgets.py
+++ b/Lib/test/test_tkinter/test_widgets.py
@@ -915,9 +915,7 @@ class TextTest(AbstractWidgetTest, unittest.TestCase):
         widget = self.create()
         self.checkParam(widget, 'tabs', (10.2, 20.7, '1i', '2i'))
         self.checkParam(widget, 'tabs', '10.2 20.7 1i 2i',
-                        expected=(10.2, 20.7, '1i', '2i')
-                                 if get_tk_patchlevel(self.root) >= (8, 6, 14)
-                                 else ('10.2', '20.7', '1i', '2i'))
+                        expected=(10.2, 20.7, '1i', '2i'))
         self.checkParam(widget, 'tabs', '2c left 4c 6c center',
                         expected=('2c', 'left', '4c', '6c', 'center'))
         self.checkInvalidParam(widget, 'tabs', 'spam',

--- a/Lib/test/test_ttk/test_widgets.py
+++ b/Lib/test/test_ttk/test_widgets.py
@@ -29,20 +29,14 @@ class StandardTtkOptionsTests(StandardOptionsTests):
 
     def test_configure_padding(self):
         widget = self.create()
-        if get_tk_patchlevel(self.root) < (8, 6, 14):
-            def padding_conv(value):
-                self.assertIsInstance(value, tuple)
-                return tuple(map(str, value))
-        else:
-            padding_conv = None
-        self.checkParam(widget, 'padding', 0, expected=(0,), conv=padding_conv)
-        self.checkParam(widget, 'padding', 5, expected=(5,), conv=padding_conv)
+        self.checkParam(widget, 'padding', 0, expected=(0,))
+        self.checkParam(widget, 'padding', 5, expected=(5,))
         self.checkParam(widget, 'padding', (5, 6),
-                        expected=(5, 6), conv=padding_conv)
+                        expected=(5, 6))
         self.checkParam(widget, 'padding', (5, 6, 7),
-                        expected=(5, 6, 7), conv=padding_conv)
+                        expected=(5, 6, 7))
         self.checkParam(widget, 'padding', (5, 6, 7, 8),
-                        expected=(5, 6, 7, 8), conv=padding_conv)
+                        expected=(5, 6, 7, 8))
         self.checkParam(widget, 'padding', ('5p', '6p', '7p', '8p'))
         self.checkParam(widget, 'padding', (), expected='')
 

